sources/packages/api/src-sql/Applications/Create-Application.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS applications (
```

Comment:
Are these going to be Applications in Draft or Fully Complete and Submitted Applications? If it can be either, then Applications is a good table name. If not, then Draft_Applications or Submitted_Applications may be better names

Reply:
I believe that we are going to keep the draft and submitted in the same table, but right now we have no clue about it.
Right now the only goal in have something concrete to prove that we can save the formio data to our DB and retrieve it in an easy way to repopulate the form.